src/modules/colorPicker/ColorPickerUI/Views/MainView.xaml

```diff
@@ -22,26 +22,70 @@
+                <Grid Margin="2" Visibility="{Binding ShowColorName, Converter={StaticResource bool2InvertedVisibilityConverter}}">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="36"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
 
-                <Border Background="{Binding ColorBrush}"
+                    <Border Background="{Binding ColorBrush}"
                     HorizontalAlignment="Stretch"
                     VerticalAlignment="Stretch"
                     BorderBrush="{DynamicResource WindowBorderBrush}"
                     BorderThickness="1"
-                    x:Name="ColorBorder"
+                    x:Name="ColorBorderSmall"
                     CornerRadius="4"/>
 
-            <TextBlock Margin="8,5,8,8"
+                    <TextBlock Margin="8,5,8,8"
                        FontSize="16"
                        FontWeight="SemiBold"
                        Foreground="{DynamicResource PrimaryForegroundBrush}"
                        Grid.Column="1"
                        Text="{Binding ColorText}"/>
+                </Grid>
+
+                <!--color format and color name - two lines-->
+                <Grid Margin="2" Visibility="{Binding ShowColorName, Converter={StaticResource bool2VisibilityConverter}}">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="26"/>
+                        <RowDefinition Height="26"/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="52"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
+
+                    <Border Background="{Binding ColorBrush}"
+                        HorizontalAlignment="Stretch"
+                        Grid.RowSpan="2"
+                        VerticalAlignment="Stretch"
+                        BorderBrush="{DynamicResource WindowBorderBrush}"
+                        BorderThickness="1"
+                        Margin="1"
+                        x:Name="ColorBorderBig"
+                        CornerRadius="4"/>
+
+                    <TextBlock Margin="8,5,6,0"
+                           FontSize="16"
+                           Grid.Row="0"
+                           FontWeight="SemiBold"
+                           Foreground="{DynamicResource PrimaryForegroundBrush}"
+                           Grid.Column="1"
+                           Text="{Binding ColorText}"/>
+
+                    <TextBlock Margin="8,0,8,2"
+                           Height="26"
+                           Grid.Row="1"
+                           Grid.Column="1"
+                           Grid.ColumnSpan="2"
+                           FontSize="16"
+                           FontWeight="SemiBold"
+                           Foreground="{DynamicResource PrimaryForegroundBrush}"
```

Comment:
Maybe SecondaryForegroundBrush would be better here? The actual color format is the primary value we want to show, the name is secondary?

Reply:
![image](https://user-images.githubusercontent.com/11967522/101983137-343c2b80-3c79-11eb-8439-0afefa0f20cd.png)

Isn't it too dark? Little hard to see

Reply:
Yeah, I agree. Can you use TitleBarForegrandBrush? That should work.

If contrast still not ideal, you can alqays revert back to PrimaryForeground brush and play around with the Opacity of the color name (e.g. 0.6)

Reply:
TitleBarForegroundBrush is dark - 
![image](https://user-images.githubusercontent.com/11967522/102124472-9ac37400-3e48-11eb-86af-968e4221f8bb.png)

Priamary and 60% opacity looks ok:
![image](https://user-images.githubusercontent.com/11967522/102124614-d0685d00-3e48-11eb-9b76-d0c8ff436d62.png)